hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -268,6 +281,10 @@ public void hflush() throws IOException {
     }
   }
 
+  public String getStreamID() {
```

Comment:
Is this needed ?

Reply:
Not needed here, but will need for tests. Shifted to test branch, will appear here after merging

Reply:
Didnt get this.